tests/apm/monitoring-topologyChanged-001.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+MongoDB\Driver\Monitoring\TopologyChangedEvent
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$m = create_test_manager();
+
+class MySubscriber implements MongoDB\Driver\Monitoring\SDAMSubscriber
+{
+    public function topologyChanged( \MongoDB\Driver\Monitoring\TopologyChangedEvent $event )
+    {
+        $oid = $event->getTopologyId();
+
+        $topologyDescription = $event->getNewDescription();
+        echo "type: ", $topologyDescription->getType(), "\n";
+        var_dump($topologyDescription->getServers());
+    }
+}
+
+$query = new MongoDB\Driver\Query( [] );
+$subscriber = new MySubscriber;
+
+MongoDB\Driver\Monitoring\addSubscriber( $subscriber );
+
+$cursor = $m->executeQuery( "demo.test", $query );
+?>
+--EXPECTF--
+type: Unknown
+array(%d) {%A
+}
+type: Single
```

Comment:
Did this test pass locally for you? If so, what was your `MONGODB_URI` environment variable defined as and what topology was the server/cluster it was referring to?

Reply:
Ah, it did pass locally but I realize now that my `MONGODB_URI` wasn't set correctly.